lld/test/wasm/page-size.s

```diff
@@ -0,0 +1,42 @@
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %s -o %t.o
+# RUN: wasm-ld -no-gc-sections -o %t.wasm %t.o --page-size=1
+# RUN: obj2yaml %t.wasm | FileCheck %s
+
+        .section .data.foo,"",@
+foo:
+        .int32  0xffffffff
+        .size   foo, 4
+
+        .section .text._start,"",@
+        .globl  _start
+_start:
+        .functype _start () -> (i32)
+        i32.const __wasm_first_page_end
+        end_function
+
+        .section .data.bar,"",@
+bar:
+        .int32  0x11111111
+        .size   bar, 4
+
+
+# CHECK:      - Type:            MEMORY
+# CHECK-NEXT:   Memories:
+# CHECK-NEXT:   - Flags:           [  ]
+# CHECK-NEXT:     Minimum:         0x10410
+# CHECK-NEXT:     PageSize:        0x1
+
+# CHECK:      - Type:            DATA
+# CHECK-NEXT:   Segments:
+# CHECK-NEXT:     - SectionOffset:   7
+# CHECK-NEXT:       InitFlags:       0
+# CHECK-NEXT:       Offset:
+# CHECK-NEXT:         Opcode:          I32_CONST
+# CHECK-NEXT:         Value:           1024
+# CHECK-NEXT:       Content:         FFFFFFFF11111111
+
+# RUN: llvm-objdump --disassemble-symbols=_start %t.wasm | FileCheck %s --check-prefix=CHECK-DIS
+
+# CHECK-DIS:      <_start>:
+# CHECK-DIS:          i32.const 1
```

Comment:
Probably worth running the same test without `--page-size=1` to check the default here.

Reply:
Good idea, thanks. Done.